egs/wsj/s5/steps/durmod/durmodel_train.sh

```diff
@@ -0,0 +1,109 @@
+#!/bin/bash
```

Comment:
You can assume that durmodel_train.sh is always going to be called with 3 arguments. In case the user wants to use dur_egs prepared else where you can support this using the --egs-dir option. Take a look at steps/nnet3/lstm/train.sh to see an example of this.